build/msw/wx_setup.props

```diff
@@ -12,6 +12,8 @@
     <wxCfg>
     </wxCfg>
     <wxVendor>custom</wxVendor>
+    <wxReleaseRuntimeLibrary>MultiThreadedDLL</wxReleaseRuntimeLibrary>
+    <wxDebugRuntimeLibrary>MultiThreadedDebugDLL</wxDebugRuntimeLibrary>
```

Comment:
Could we perhaps define `wxRuntimeLibs` and check below if it's `dynamic` (default value) or `static` and set these 2 automatically? It looks like it would be more consistent with the make option used currently and simpler to use.

Reply:
I think I understand what you mean here. Let me see what I can come up with...

Reply:
Forgot to mention - this is ready now.